src/index.css

```diff
@@ -0,0 +1,25 @@
+body {
+  background: rgb(34, 193, 195);
+  background: linear-gradient(0deg, rgba(34, 193, 195, 1) 0%, rgba(253, 187, 45, 1) 100%);
+}
+
+code {
+  font-family:
+    source-code-pro,
+    Menlo,
+    Monaco,
+    Consolas,
+    'Courier New',
+    monospace;
+}
+
+@media screen and (min-width: 1024px) {
+  .cards-container {
+    margin: auto;
+  }
+
+  .card-container {
+    width: 40%;
+    margin: auto;
+  }
+}
```

Comment:
- [ ] Following the requirements, you should use the same `font families` that template shows. ⚒️ 

![image](https://user-images.githubusercontent.com/79876959/147312658-cdfd8757-8da0-42b1-8093-71523888429b.png)
![image](https://user-images.githubusercontent.com/79876959/147312694-8d692868-d366-4583-809e-8173265637a5.png)

Reply:
✔️